### PR TITLE
fix relative paths in python script

### PR DESCRIPTION
**Pull request by ericwu17**
This pull request fixes the relative path in the `fix-chars.py` python script, which should allow the python script to be ran by the following command 

```
python3 data-raw/fix-chars.py
```

### CODE REVIEW

**Comment by wjakethompson** [comment]
Thanks! I'm able to run the script now, but I'm getting another error in the process of running:

```
python3 data-raw/fix-chars.py
Traceback (most recent call last):
  File "/Users/jakethompson/Documents/GIT/packages/taylor/data-raw/fix-chars.py", line 38, in <module>
    replaceChars(path)
  File "/Users/jakethompson/Documents/GIT/packages/taylor/data-raw/fix-chars.py", line 8, in replaceChars
    lyrics = f.read()
             ^^^^^^^^
  File "<frozen codecs>", line 322, in decode
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x80 in position 3131: invalid start byte
```

The script is partially working, as I can see that a couple of the lyrics txt files have been changed. But I don't think all of them, as "The Black Dog" has not been changed, and that was the original example in #44. Any thoughts on what is causing the script to fail?

**Comment by ericwu17** [comment]
Hmmm, do you have any binary files in the folders that contain the text files for the lyrics?

The python script will start in the `data-raw/lyrics` directory, and then it will go through each folder assuming that the folder contains text files of an album. If there is a folder that contains an additional binary file(s), or additional folders other than the folders with album data, then I would expect to see the error.

I made a new commit that changes the python script to only act on `.txt` files. Hopefully if you run the script now it should ignore binary files and only operate on the `.txt` files.

File: data-raw/fix-chars.py
Comment (ericwu17) [inline data-raw/fix-chars.py:1]: I didn't notice when committing, but it looks like I also converted some indentation from spaces to tabs.

**Comment by wjakethompson** [comment]
That worked! I was able to run the script and update all of the lyrics. I also added a step to my process when adding a new album to run the script so that hopefully this issue doesn't pop up again in the future. Thanks!

Closes #44.

**Pull request merged**